samples/nrf_desktop/src/hw_interface/buttons.c

```diff
@@ -455,6 +465,10 @@ static void init_fn(void)
 
 static bool event_handler(const struct event_header *eh)
 {
+	if (unlikely(state == STATE_ERROR)) {
```

Comment:
no need to do it - just cancel work and do not submit it on error

Reply:
ok